xbox/rest/routes/device.py

```diff
@@ -188,7 +200,7 @@ def media_command(console, command):
     return app.success()
 
 
-@routes.route('/device/<liveid>/media/Seek/<seek_position>')
+@routes.route('/device/<liveid>/media/seek/<seek_position>')
```

Comment:
While this is a reasonable change it differs from the standards.
`/media` returns a dictionary of commands -it uses the same casing as the original enum, meaning:
`PlayPause`, `Play`, `Seek`
In `/media/<command>` the MediaCommand enum member is resolved by passing in the matching-case command. `seek` and `Seek` would work - it has the MediaCommand hardcoded.. it would fail for `/media/<command>` if lowercase.

Long story short: A dictionary should be used, that translates command.lower() to the appropriate `MediaCommand`.
See: https://github.com/hunterjm/xbox-smartglass-rest-python/blob/feature/authentication/xbox/rest/routes/device.py#L196

Reply:
PS: can you enable contributor changes please?